cmd/jujud/agent/machine/manifolds.go

```diff
@@ -547,19 +547,17 @@ func commonManifolds(config ManifoldsConfig) dependency.Manifolds {
 			Logger:            loggo.GetLogger("juju.worker.migrationminion"),
 		}),
 
-		// We start clock updaters for every controller machine to ensure the
-		// lease clock is updated monotonically and at a rate no faster than
-		// real time.
-		// In practice, dependency on the Raft forwarder means that this worker
-		// will only ever be running on the Raft leader node.
-		leaseClockUpdaterName: globalclockupdater.Manifold(globalclockupdater.ManifoldConfig{
-			Clock:            config.Clock,
-			LeaseManagerName: leaseManagerName,
-			RaftName:         raftForwarderName,
```

Comment:
There was no need for a synthetic dependency on the forwarder, because it is just a proxy for running on the leader. We can use `ifRaftLeader` directly for this.